### ISSUE TITLE
Issues when running `start` script

### ISSUE THREAD

**Issue by alechp** [Mozart409/gatsby-custom-tailwind#9]
### Versions
![Screen Shot 2020-05-26 at 2 42 10 PM](https://user-images.githubusercontent.com/3058939/82953127-15857a00-9f5f-11ea-969d-f313f10260ab.png)

![Screen Shot 2020-05-26 at 2 42 31 PM](https://user-images.githubusercontent.com/3058939/82953162-2209d280-9f5f-11ea-8355-53812f5c922f.png)

### Error(s)
![Screen Shot 2020-05-26 at 2 43 10 PM](https://user-images.githubusercontent.com/3058939/82953275-52517100-9f5f-11ea-9772-57df43ce0f87.png)

![Screen Shot 2020-05-26 at 2 43 28 PM](https://user-images.githubusercontent.com/3058939/82953269-4d8cbd00-9f5f-11ea-8795-53c3934c66d7.png)

**Comment by alechp** [comment]
Trying suggestions listed here, will update when finished:
https://github.com/gatsbyjs/gatsby/issues/23062

**Comment by alechp** [comment]
### TLDR
Fixed it removing `graphql: 15.0.0` from dependencies

### Debug Notes
> [Debugging - Gatsby-custom-tailwind.pdf](https://github.com/Mozart409/gatsby-custom-tailwind/files/4685262/Debugging.-.Gatsby-custom-tailwind.pdf)

**Comment by Mozart409** [comment]
Removed graphql https://github.com/Mozart409/gatsby-custom-tailwind/commit/65844173dfc207406bc56364196f1574eb48f7cb

**Comment by Mozart409** [comment]
> 
> ### TLDR
> 
> Fixed it removing `graphql: 15.0.0` from dependencies
> ### Debug Notes
> 
> > [Debugging - Gatsby-custom-tailwind.pdf](https://github.com/Mozart409/gatsby-custom-tailwind/files/4685262/Debugging.-.Gatsby-custom-tailwind.pdf)

Wow. Thank you for your analysis. I removed graphql dependency and tested the dev and prod build. Both work now without any errors.

**Comment by alechp** [comment]
> Wow. Thank you for your analysis. I removed graphql dependency and tested the dev and prod build. Both work now without any errors.

Happy to help! Thanks for the excellent template

**Comment by Mozart409** [comment]
> > Wow. Thank you for your analysis. I removed graphql dependency and tested the dev and prod build. Both work now without any errors.
> 
> 
> 
> Happy to help! Thanks for the excellent template

Thanks. Have a great weekend. 😊
